modules/light-clients/08-wasm/types/msgs.go

```diff
@@ -47,3 +50,46 @@ func (m MsgPushNewWasmCode) GetSigners() []sdk.AccAddress {
 func (msg MsgPushNewWasmCode) GetSignBytes() []byte {
 	return sdk.MustSortJSON(ModuleCdc.MustMarshalJSON(&msg))
 }
+
+// NewMsgUpdateWasmCodeId creates a new MsgUpdateWasmCodeId instance
+//
+//nolint:interfacer
+
+// Route Implements Msg.
+func (msg MsgUpdateWasmCodeId) Route() string { return ModuleName }
+
+// Type Implements Msg.
+func (msg MsgUpdateWasmCodeId) Type() string { return TypeMsgUpdateWasmCodeId }
+
+// TODO: add tests for MsgUpdateWasmCodeId
+
+func NewMsgUpdateWasmCodeId(signer string, codeId []byte, clientId string) *MsgUpdateWasmCodeId {
+	return &MsgUpdateWasmCodeId{
+		Signer:   signer,
+		CodeId:   codeId,
+		ClientId: clientId,
+	}
+}
+
+func (m MsgUpdateWasmCodeId) ValidateBasic() error {
```

Comment:
Can't we validate the `ClientId` here using `ClientIdentifierValidator` function for safety?

```go
// ClientIdentifierValidator is the default validator function for Client identifiers.
// A valid Identifier must be between 9-64 characters and only contain alphanumeric and some allowed
// special characters (see IsValidID).
func ClientIdentifierValidator(id string) error {
	return defaultIdentifierValidator(id, 9, DefaultMaxCharacterLength)
}
```

Reply:
Added the validation